doc/README.Win32.md

```diff
@@ -1,3 +1,55 @@
-Win32 used to build with Visual Studio 6, but we now use cmake.
+# Introduction
 
-This file needs to be adopted by a windows expert developer.
+A recent version of Visual Studio is needed to handle CMake projects.
```

Comment:
By this are you referring to [the built-in CMake support introduced in Visual Studio 2017](https://devblogs.microsoft.com/cppblog/cmake-support-in-visual-studio/)?

If so, please note that:

- you should say "Visual Studio 2017 or later", not just "a recent version";
- you don't necessarily need VS 2017 or later to build, but if you have an earlier version, you'll have to run CMake yourself to generate the .prj and .sln files for VS.

Note also that there is no need to run the Visual Studio IDE to build libpcap (or tcpdump or Wireshark or...) - you can just use msbuild.

Reply:
Updated.